### ISSUE TITLE
Add colors to plugin

### ISSUE THREAD

**Issue by eyalk11** [gravelBridge/AutoGPT-Discord#3]
May need to convert ANSI to rich text in discord... probably possible.

**Comment by CTHULHUCTHULHU** [comment]
not really, discord lacks support for such - you can cheat your way into colored text but in a very nasty way. I would just wait for full .md support
